lib/Table/TableTimeVaryingStratum.ts

```diff
@@ -0,0 +1,30 @@
+import { computed } from "mobx";
+import CsvCatalogItem from "../Models/CsvCatalogItem";
+import LoadableStratum from "../Models/LoadableStratum";
+import StratumFromTraits from "../Models/StratumFromTraits";
+import CsvCatalogItemTraits from "../Traits/CsvCatalogItemTraits";
+import DiscreteTimeTraits from "../Traits/DiscreteTimeTraits";
+import TableColumnType from "./TableColumnType";
+
+export default class TableTimeVaryingStratum extends LoadableStratum(
+  CsvCatalogItemTraits
+) {
+  constructor(readonly catalogItem: CsvCatalogItem) {
```

Comment:
Maybe `CsvCatalogItem` here can be more general too? If not, this stratum should be renamed to indicate it is CSV-specific.